django/contrib/auth/hashers.py

```diff
@@ -322,53 +323,76 @@ def encode(self, password, salt):
             parallelism=self.parallelism,
             hash_len=argon2.DEFAULT_HASH_LENGTH,
             type=argon2.low_level.Type.I,
+            version=self.version,
```

Comment:
I would directly check against `argon2.low_level.ARGON2_VERSION`. What I want to prevent is a new argon version increasing the version number and we have to issue a new django release to take advantage of that.